### ISSUE TITLE
Attachment Handling Issue

### ISSUE THREAD

**Issue by d4em0nsp4wn** [thunderbird-conversations/thunderbird-conversations#1882]
* **Thunderbird version:** 102.0.1
* **Thunderbird Conversation version:** 4.0.19
* **Operating System:** Windows 10 Enterprise
 
**What did you do?**  Clicked on Attachment (not the Download Attachment Arrow)

**What did you expect to happen?** Open File or Prompt to Open the attachment

**What actually happened?**  Attachment opened and stored file on Desktop; not a temporary path or provide a prompt to ask where to save it. 

It is really inconvenient to have files cluttering my desktop; I should be able to designate an folder to save the attachment upon choosing to click on it or the native behavior of the attachment opening from a temporary folder location be maintained instead of forcing all the files to be saved on the desktop.

**Comment by Standard8** [comment]
Conversations implements the same behaviour as native Thunderbird with respect to attachment handling. Handling download of attachments differently is not part of Conversation's scope, nor something I'm willing to add to it. Sorry, but you'll need to file an issue with Thunderbird themselves - https://bugzilla.mozilla.org/

**Comment by d4em0nsp4wn** [comment]
So I did a bit more digging; in Thunderbird -> Settings -> Files & Attachments; the Conversation add-on is not honoring the setting "Always ask me where to save files" when clicking on the attachment itself to simply open it; but it is honoring the setting "Save Files to"; once I changed that setting; Conversation is storing attachments correctly in a temporary location; even though my default option is still "Always ask me where to save files".  While not a bug; it is a confusion in documentation. But I appreciate you taking the time to respond so quickly! Have a great one.
